public/static/js/spawned_hosts.js

```diff
@@ -484,7 +484,8 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q',
       mciSpawnRestService.spawnHost(
         $scope.spawnInfo, {}, {
           success: function (resp) {
-            $window.location.reload();
+            window.location.href = "/spawn";
```

Comment:
I think `$window` is AngularJS's `window`
(https://docs.angularjs.org/api/ng/service/$window)

Reply:
Ohh I was confused this is what the code looked like before your change and because window does work 🤔 Maybe that was a mistake or bad style though.

Reply:
Also in my next commit I typed the message as "use $window" and now it's different so magic is all around us haha

Reply:
$window and window _should_ work about the same.
If so `$window.location.href = "/spawn";` would also work

Reply:
Did you enter the commit from the command line with the `-m, --message` option?
I wonder if bash substituted the contents of the `window` variable with the empty string in the commit message.